### PR TITLE
[zh] Sync /traffic-management/egress/egress-gateway/

### PR DESCRIPTION
**Pull request by windsonsea**
Sync with a big file in en #14473 

```
content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md [hunk 1]
```diff
@@ -33,20 +33,55 @@ Egress Gateway 允许您将 Istio 的功能（例如，监视和路由规则）
...
+*   将 `SOURCE_POD` 环境变量设置为源 Pod 的名称：
+
+    {{< text bash >}}
+    $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})
+    {{< /text >}}
+
+    {{< warning >}}
+    此任务中的指令在 `default` 命名空间中为 Egress Gateway 创建目标规则。
+    并假设客户端 `SOURCE_POD` 也在 `default` 命名空间中运行。如果没有，
+    目标规则则不会在[目标规则查找路径](/zh/docs/ops/best-practices/traffic-management/#cross-namespace-configuration)上找到，
```
Comment (hanxiaop) [inline content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md:66]:
```suggestion
    目标规则将不会在[目标规则查找路径](/zh/docs/ops/best-practices/traffic-management/#cross-namespace-configuration)上找到，
```

File: content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md [hunk 2]
```diff
@@ -125,83 +160,163 @@ Egress Gateway 节点，用它引导所有的出站流量，可以使应用节
...
+      namespaces:
+        from: Same
+EOF
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+4)  配置路由规则，将流量从 Sidecar 导向到 Egress Gateway，再从 Egress Gateway 导向到外部服务：
```
Comment (hanxiaop) [inline content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md:232]:
```suggestion
4)  配置路由规则，将流量从边车导向到 Egress Gateway，再从 Egress Gateway 导向到外部服务：
```

File: content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md [hunk 3]
```diff
@@ -440,58 +715,115 @@ Egress Gateway 的出站流量。为了测试网络策略，首先创建一个 `
...
-        ports:
-        - protocol: UDP
-          port: 53
-      - to:
-        - namespaceSelector:
-            matchLabels:
-              istio: system
-    EOF
-    {{< /text >}}
+{{< warning >}}
```
Comment (hanxiaop) [inline content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md:755]: indent error?

File: content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md [hunk 4]
```diff
@@ -440,58 +715,115 @@ Egress Gateway 的出站流量。为了测试网络策略，首先创建一个 `
...
+[网络政策](https://kubernetes.io/zh-cn/docs/concepts/services-networking/network-policies/)由您的
```
Comment (hanxiaop) [inline content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md:756]:
```suggestion
[网络政策](https://kubernetes.io/zh-cn/docs/concepts/services-networking/network-policies/)由
```

File: content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md [hunk 5]
```diff
@@ -33,20 +33,55 @@ Egress Gateway 允许您将 Istio 的功能（例如，监视和路由规则）
...
-*   [启用 Envoy 访问日志](/zh/docs/tasks/observability/logs/access-log/#enable-envoy-s-access-logging)
+## 开始之前 {#before-you-begin}
 
-{{< warning >}}
-此任务中的指令在 `default` 命名空间中为 Egress gateway 创建目标规则。
-并假设客户端 `SOURCE_POD` 也在 `default` 命名空间中运行。如果没有，
-目标规则则不会在[目标规则查找路径](/zh/docs/ops/best-practices/traffic-management/#cross-namespace-configuration)上找到，
-并且客户端请求将失败。
-{{< /warning >}}
+*   参照[安装指南](/zh/docs/setup/)中的说明搭建 Istio。
```
[the remark this answers is not in the document]
Comment (hanxiaop) [reply content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md:40]: Makes sense. Looks like we only have space between english words and punctuations.

File: content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md (hunk 2 above)
Comment (windsonsea) [reply content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md]: We have 718 `sidecar` and 24 `边车` in current zh folder

File: content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md (hunk 3 above)
Comment (windsonsea) [reply content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md]: updated

File: content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md (hunk 2 above)
Comment (hanxiaop) [reply content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md]: I was thinking Sidecar may refer to the Sidecar resource https://istio.io/latest/docs/reference/config/networking/sidecar/. However if we are using it in multiple places, I think it should be good. Thanks!

**Review by hanxiaop — APPROVED** [review]

File: content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md (hunk 2 above)
Comment (windsonsea) [reply content/zh/docs/tasks/traffic-management/egress/egress-gateway/index.md]: ok, all right, it is changed to `边车` here by following suggestions.

**Comment by windsonsea** [comment]
cc @wilsonwu

**Review by zirain — APPROVED** [review]

**Review by kfaseela — APPROVED** [review]

**Pull request merged**